### PR TITLE
add the "reorder" command

### PR DESCRIPTION
**Pull request by upseliseev**
add code for the reorder command. 
also tweak some of the console messages

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -64,9 +65,73 @@ static async Task Main(string[] args)
...
+                else return 0; 
+            });
+
+            Random random = new Random(); 
+            Int32 index1;
+            Int32 index2;
+            // looped to make sure we don't get the same value for both indices, since such a switch is not informative 
+            // and does not benefit anyone, and is mostly a test of this extension than of the client 
+            do {
+                index1 = random.Next(cards.Count);
```
Comment (asugaya-apx) [inline Program.cs:108]: INFINITE LOOP DETECTED!

**Review by samUpskill — APPROVED** [review]
tis fine

**Review by asugaya-apx — COMMENTED** [review]
Nice! YAY EXTENSIONS AND QA! :) gj sofia

File: Program.cs (hunk 1 above)
Comment (asugaya-apx) [reply Program.cs:108]: (it's fine)

Comment (upseliseev) [reply Program.cs:108]: you can cill it with `CTRL+C`

Comment (asugaya-apx) [reply Program.cs:108]: tru

**Pull request merged**